app/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'com.android.application'
+apply plugin: 'com.jakewharton.butterknife'
```

Comment:
Why do you use this plugin? It's needed if you use butteknife in a library only.